cloudigrade/account/tests/test_tasks.py

```diff
@@ -137,8 +145,38 @@ def test_copy_ami_snapshot_retry_on_ownership_not_verified(self, mock_aws):
         mock_aws.add_snapshot_ownership.side_effect = \
             AwsSnapshotNotOwnedError()
 
-        with patch.object(copy_ami_snapshot, 'retry') as mock_retry:
+        with patch.object(tasks, 'create_volume') as mock_create_volume,\
+                patch.object(copy_ami_snapshot, 'retry') as mock_retry:
             mock_retry.side_effect = Retry()
-
             with self.assertRaises(Retry):
                 copy_ami_snapshot(mock_arn, mock_image_id, mock_region)
+            mock_create_volume.assert_not_called()
+
+    @patch('account.tasks.aws')
+    def test_create_volume_success(self, mock_aws):
+        """Assert that the volume create task succeeds."""
+        ami_id = util_helper.generate_dummy_image_id()
+        snapshot_id = util_helper.generate_dummy_snapshot_id()
+        zone = settings.HOUNDIGRADE_AVAILABILITY_ZONE
+
+        mock_volume = util_helper.generate_mock_volume()
+        mock_aws.create_volume.return_value = mock_volume.id
+
+        create_volume(ami_id, snapshot_id)
+
+        mock_aws.create_volume.assert_called_with(snapshot_id, zone)
```

Comment:
In this case, though, there is nothing returned from `create_volume` because it's a Celery task. The test for the `aws.create_volume` helper that's being called by this task, though, does have an assertion like this. See `test_create_volume_snapshot_ready`.

Reply:
Yep, I just lost track of which file I was in.